schema/rdl_schema_0.1.json

```
@@ -144,12 +132,18 @@
     "sources": {
       "title": "Sources",
       "type": "array",
-      "description": "Related resources that serve as sources of information or predictions that contribute to the creation of the dataset.",
+      "description": "Sources of information or predictions that contribute to the creation of the dataset.",
       "items": {
         "$ref": "#/$defs/Source"
       },
       "minItems": 1,
       "uniqueItems": true
+    },
+    "temporal_resolution": {
+      "title": "Temporal resolution",
+      "type": "string",
+      "description": "The minimum period of time resolvable in this dataset, in ISO8601 duration format. This field must be used if the temporal resolution of all the resources in the dataset are the same. If they are not this field must not be used and `resource.temporal_resolution` used instead.",
```

I don't think we need that bit since the semantics are the minimum period of time resolvable.

```suggestion
      "description": "The minimum period of time resolvable in this dataset, in ISO8601 duration format.",
```